modules/ROOT/pages/6.4-release-notes.adoc

```diff
@@ -0,0 +1,1252 @@
+
+To prevent this from happening, {productname} made improvements to the Template plugin that now sanitises content when retrieving preview content, and again before running `editor.dom.create` on it during the template insertion action. By doing this, all preview content will now match the template when it is inserted into the editor, which removed the possibility of any XSS issues arrising.
+
+In {productname} 6.4, Templates are now parsed before preview and insertion to make preview consistent with inserted template content and prevent XSS from occuring.
+
+// https://notion.so/tinycloud/Four-words-to-live-by-96e307110aaa4137b94f9ba12874d480
+
+=== Pressing backspace in an empty line now preserves formatting in a previous empty line
+
+// https://ephocks.atlassian.net/browse/TINY-9454
+In previous versions of {productname}, pressing `backspace` in an empty line would not preserve previously applied formatting to the previous line.
+
+As a consequence, when backspacing from an empty block from another empty block within {productname}, the editor would lose the preserved formatting for the previous block element.
+Its worth noting that `_this action is only excpected for when content is inserted into a `br[data-mce-bogus]` empty block. This is because the br[data-mce-bogus] tag is programmed to reset the formatting to the default setting for a empty block such as the `br`.
+
+[TIP]
+To avoid this, it is *important* to use a `backspace` instead of an `enter key` when moving from one empty block to another. This will ensure that the formatting of the previous block is preserved and the `br[data-mce-bogus]` tag is not inserted.
+
+To fix this issue, {productname} added improvements that now allow merge blocks to filter all block element decentants, such as the `br[data-mce-bogus]` block element, to ensure preservation of previously applied formatting is not removed from other block elements.
+
+In {productname} 6.4, pressing backspace in an empty line now preserves formatting in a previous empty line.
+
+// https://notion.so/tinycloud/Four-words-to-live-by-96e307110aaa4137b94f9ba12874d480
+
+=== Pressing enter inside the inputfontsize input would not move the focus back into the editor content
+
+// https://ephocks.atlassian.net/browse/TINY-9598
+During development for the new `fontsizeinput` toolbar item, an issue was found that by pressing enter inside the input field would not move the `focus` back into the {productname} editor body. 
+
+As a consequence, any changes made to the font size within the new toolbar input field, would result in the user having no way to use the new font size within the editor.
+
+After some improvements to the `fontsizeinput` toolbar item, pressing enter in the input field now sets the focus back to the {productname} editor viewport, which allows the new font size to be used. 
+
+In {productname} 6.4, pressing enter inside the `inputfontsize` input now returns the focus back to the editor content body.
```

Comment:
Is it `inputfontsize` or `fontsizeinput`? Something is wrong here.

Reply:
@spocke So if we change all instances of `inputfontsize` and replace with `inputfontinput` including the heading, we should be ok with the above suggested documentation?

Reply:
This has been corrected.